.github/workflows/acd-containers-build.yml

```diff
@@ -28,15 +28,11 @@ jobs:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v3
 
-      # Runs a single command using the runners shell
-      - name: Run a one-line script
-        run: echo Hello, world!
-
       # Runs a set of commands using the runners shell
       - name: Run a multi-line script
         run: |
           npm install -g markdownlint-cli
-          npm ci --force
+          npm install -force
```

Comment:
Not sure I fully understand the difference between the clean install and regular install, but appears to be working again and the pages load. Thanks.

Reply:
Clean install does deeper analysis of chained dependencies which eventually we want to get back to but there is a lot of compatibility issues between Gatsby and Gatsby plugins that need to be worked through.  Normally we keep the dependency list in package.json as short as possible to help with this but a lot has been included and they have conflicting version declarations.  I'll work through that in the background as it doesn't impact the gh-pages deployment of the documentation.